operator/pkg/name/name.go

```diff
@@ -86,6 +86,15 @@ const (
 	IstioOperatorCustomResourceName ComponentName = "IstioOperatorCustomResource"
 )
 
+var IstioComponentSuccessIcons = map[ComponentName]string{
+	IstioBaseComponentName: "⛵️",
+	PilotComponentName:     "🧠",
+	CNIComponentName:       "🚦",
+	ZtunnelComponentName:   "🔒",
+	IngressComponentName:   "🚪",
+	EgressComponentName:    "✋",
```

Comment:
Maybe use similar or same icons for ingress and egress?

Reply:
I thought about 🛫 (plane takeoff/leave) for egress and 🛬 (plane land/arrive) for ingress.

Reply:
@linsun, how do you feel about using the planes?